addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -242,6 +243,22 @@ public void registerAutomationJob(AutomationJob job) {
     public void unregisterAutomationJob(AutomationJob job) {
         this.jobs.remove(job.getType());
         this.sortedJobs.remove(job);
+
+        Set<Integer> keys = plans.keySet();
+        for (Integer key : keys) {
+            System.out.println(key);
```

Comment:
Use a logger if output is really needed

Reply:
ohh I am sorry., forgot to remove that. It was just for my own debugging :)

Reply:
I guess I would also have to learn the logger
because working with these print statements is actually a very annoying thing :p

Reply:
A logger isn’t much different, but never hurts to learn something news.

Reply:
Yes